package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@semantic-release/github",
+  "description": "Set of semantic-release plugins for publishing a Github release",
+  "version": "0.0.0-development",
+  "author": "Pierre Vanduynslager (https://twitter.com/@pvdlg_)",
+  "bugs": {
+    "url": "https://github.com/semantic-release/github/issues"
+  },
+  "config": {
+    "commitizen": {
+      "path": "cz-conventional-changelog"
+    }
+  },
+  "contributors": [
+    "Stephan Bönnemann <stephan@boennemann.me> (http://boennemann.me)",
+    "Gregor Martynus (https://twitter.com/gr2m)"
+  ],
+  "dependencies": {
+    "@semantic-release/error": "^2.1.0",
+    "debug": "^3.1.0",
+    "fs-extra": "^4.0.2",
+    "git-url-parse": "^7.0.1",
+    "github": "^12.0.0",
+    "p-each-series": "^1.0.0"
+  },
+  "devDependencies": {
+    "ava": "^0.23.0",
+    "clear-module": "^2.1.0",
+    "codecov": "^3.0.0",
+    "commitizen": "^2.9.6",
+    "cz-conventional-changelog": "^2.0.0",
+    "eslint": "^4.10.0",
+    "eslint-config-prettier": "^2.5.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-node": "^5.2.0",
+    "eslint-plugin-prettier": "^2.3.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "nock": "^9.1.0",
+    "nyc": "^11.2.1",
+    "prettier": "~1.8.0",
+    "rimraf": "^2.5.0",
+    "semantic-release": "^9.1.0",
+    "sinon": "^4.0.0"
+  },
+  "engines": {
+    "node": ">=4"
```

Comment:
we just leave this because of something with Yarn, right? Because `index.js` will fail with Node < 8

Reply:
Yes. Not a great solution....
I wish we could come up with something better